pandas/core/internals/managers.py

```diff
@@ -184,7 +184,7 @@ def set_axis(self, axis, new_labels):
 
         self.axes[axis] = new_labels
 
-    def rename_axis(self, mapper, axis, copy: bool = True, level=None):
+    def rename_axis(self, mapper, axis: int, copy: bool = True, level=None):
```

Comment:
return type, mapper and level

Reply:
looks like `level` may be inconsistent with the caller's signature, so punting for now.  xref #32349.